### ISSUE TITLE
removeItemButton is not adding a 'x' to single select

### ISSUE THREAD

**Issue by ganeshkantu** [NetanelBasal/nb-choices#2]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Feature request
[ ] Documentation issue or request
</code></pre>

## Current behavior

The `removeItemButton` option which is set to true by default does not add a 'x' to single selects. Works correctly for multi-selects and text.

## Expected behavior

If the `removeItemButton` option is set to true, an option to clear the value in single select should be added.

## Minimal reproduction of the problem with instructions

The demo has this problem. 

## What is the motivation / use case for changing the behavior?

This option works correctly in choices.js. This wrapper should replicate that behavior.

## Environment

<pre><code>
Angular version: 5.1.0

Browser:
- [x] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
Others:

</code></pre>

**Comment by NetanelBasal** [comment]
Hey,

The single select does not support the x button.

**Comment by ganeshkantu** [comment]
Hey - thanks for the lib btw!

Take a look at the demo page for choices.js - https://joshuajohnson.co.uk/Choices/
Scroll down to single select - there is a demo with a remove button which is used to clear the selection.

**Comment by NetanelBasal** [comment]
Yes, because design changes, I dropped the support, for now, I don't have time right now, so if you want this, I would love to see a pull request.

**Comment by ganeshkantu** [comment]
Cool. Will take a look.
